BetterShardsBukkit/src/main/java/vg/civcraft/mc/bettershards/external/MercuryManager.java

```diff
@@ -65,54 +65,58 @@ public void teleportPlayer(UUID uuid, Portal p, Object ... data) {
 	 */
-	public void teleportPlayer(String server, UUID uuid, TeleportInfo info) {
+	public static void teleportPlayer(String server, UUID uuid, TeleportInfo info) {
 		MercuryAPI.sendMessage(server, "teleport|command|" + uuid.toString() + "|" + info , "BetterShards");
 	}
 	
-	public void teleportPlayer(String server, UUID playerToTeleportUUID, UUID targetPlayerUUID) {
+	public static void teleportPlayer(String server, UUID playerToTeleportUUID, UUID targetPlayerUUID) {
 		MercuryAPI.sendMessage(server, "teleport|command|" + playerToTeleportUUID + "|" + targetPlayerUUID , "BetterShards");
 	}
 	
-	public void teleportPlayer(String server, UUID playerToTeleportUUID, String x, String y, String z) {
+	public static void teleportPlayer(String server, UUID playerToTeleportUUID, String x, String y, String z) {
 		MercuryAPI.sendMessage(server, "teleport|command|" + playerToTeleportUUID + "|" + x + "|" + y +"|" + z , "BetterShards");
 	}
 	
-	public void teleportPlayer(String server, UUID playerToTeleportUUID, String x, String y, String z, String world) {
+	public static void teleportPlayer(String server, UUID playerToTeleportUUID, String x, String y, String z, String world) {
 		MercuryAPI.sendMessage(server, "teleport|command|" + playerToTeleportUUID + "|" + world + "|" + x + "|" + y +"|" + z, "BetterShards");
 	}
 
-	private void registerMercuryChannels() {
+	private static void registerMercuryChannels() {
 		MercuryAPI.addChannels("BetterShards");
 	}
 
-	public void sendBedLocation(BedLocation bed) {
+	public static void sendBedLocation(BedLocation bed) {
 		String info = bed.getUUID().toString() + "|" + bed.getServer() + "|" + bed.getTeleportInfo();
 		MercuryAPI.sendGlobalMessage("bed|add|" + info, "BetterShards");
 	}
 	
-	public void notifyRandomSpawn(String server, UUID player) {
+	public static void notifyRandomSpawn(String server, UUID player) {
 		MercuryAPI.sendMessage(server, "teleport|randomspawn|" + player.toString(), "BetterShards");
 	}
 
-	public void removeBedLocation(BedLocation bed) {
+	public static void removeBedLocation(BedLocation bed) {
 		MercuryAPI.sendGlobalMessage("bed|remove|" + bed.getUUID().toString(), "BetterShards");
 	}
 
-	public void sendBungeeUpdateMessage() {
+	public static void sendBungeeUpdateMessage() {
 		// Remove Server is just the message being sent.
 		// Bungee ends up calling the db to get all the update tables.
 		MercuryAPI.sendGlobalMessage("removeServer", "BetterShards");
 	}
 	
-	public void sendPortalJoin(Portal main, Portal con) {
+	public static void sendPortalJoin(Portal main, Portal con) {
 		MercuryAPI.sendGlobalMessage("portal|connect|"+main.getName()+"|"+con.getName(), "BetterShards");
 	}
 	
-	public void removePortalJoin(Portal main) {
+	public static void removePortalJoin(Portal main) {
 		MercuryAPI.sendGlobalMessage("portal|remove|"+main.getName(), "BetterShards");
 	}
 	
-	public void sendPlayerCount(int count) {
+	public static void sendPlayerCount(int count) {
```

Comment:
Because it is static. Doesnt depend on an instance of the class and is just an API, so passing around an instance is pointless and it might as well be static. Not really related to the other changes, just something I stumbled across.